File: 1929-concatenation-of-array/1929-concatenation-of-array.py
class Solution(object):
    def getConcatenation(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        arr = []
        for i in range(0,len(nums)):
            arr.append(nums[i])
        for i in range(0,len(nums)):
            arr.append(nums[i])
        return arr

    

File: 1929-concatenation-of-array/1929-concatenation-of-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {
    let n = nums.length
    let arr = new Array((n*2));
    for(let i =0;i<nums.length;i++){
        arr[i] = nums[i]
        arr[n+i] = nums[i]
    }
    return arr
};